docs/server.rst

```diff
@@ -35,7 +35,7 @@ the broker.
 Operating System
 ----------------
 
-The first thing you need is a recent (>= 2.6.36) Linux kernel that supports L2TPv3
+The first thing you need is a recent (>= 3.9) Linux kernel that supports L2TPv3
```

Comment:
In [the issue](https://github.com/wlanslovenija/tunneldigger/issues/126) you say "3.9 and older" does not work, but this one says "3.9 and newer" will work. So does 3.9 itself work or not? The two statements are contradiction each other in that regard.

Reply:
The issue also claims that 4.19.75 and 5.2.17 don't work even though you said elsewhere those are the first fixed version, so this is probably just an issue of you using `<= version` instead of `< version` in the issue text.

Reply:
Totally agree.
I updated the issue to reflect your comments. Please check if it makes more sense that way.

Reply:
I did some more edits that IMO clarify things further, let me know what you think.
Given that the leading version number in Linux (3, 4, 5) is utterly meaningless, I'd prefer to group things by major kernel versions (x.y).

Reply:
Yes. That is much better now. Thanks!

Reply:
Actually from what I could find, the fix is in 5.4 but not in 5.3, so I adjusted the issue accordingly.

Reply:
Ah, it got backported to 5.3.1.